README.md

```diff
@@ -1,70 +1,75 @@
-# C++ Samples
-
-This repository contains C++ samples for Mech-Eye SDK.
-
-## Installation
-
-1. Download and install [Mech-Eye SDK](https://www.mech-mind.com/download/camera-sdk.html).
-2. Clone this repository to a specific folder.
-3. Configure the sample solution with CMake, open it in Visual Studio, build and run it.
-
-## Sample list
-
-There are four categories of samples: **Basic**, **Advanced**, **Util**, and **Laser**.  
-
-- The category **Basic** contains samples that are related to basic connecting and capturing.  
-- The category **Advanced** contains samples that use advanced capturing tricks.  
-- The category **Util** contains samples that get and print information and set parameters.  
-- The category **Laser** contains samples that can only be used on Mech-Eye Laser cameras.  
-
```

Comment:
- The category **UHP** contains samples that can only be used on Mech-Eye UHP cameras.